modules/op-release-notes-1-8.adoc

```diff
@@ -22,19 +22,21 @@ include::snippets/technology-preview.adoc[]
 
 
 [id="deprecated-features-1-8_{context}"]
-== Deprecated features
+== Deprecated and removed features
 
-* Support for the `tekton.dev/v1alpha1` API version, which was deprecated in {pipelines-title} GA 1.6, is planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
+* In {pipelines-title} 1.8, support for the `tekton.dev/v1alpha1` API version, which was deprecated in {pipelines-title} GA 1.6, is planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
 +
 This change affects the pipeline component, which includes the `TaskRun`, `PipelineRun`, `Task`, `Pipeline`, and similar `tekton.dev/v1alpha1` resources. The `operator.tekton.dev/v1alpha1` API version is not deprecated and still used by the Operator.
 +
 Bug fixes and support for the `tekton.dev/v1alpha1` API version are provided only through the end of the current GA 1.8 lifecycle. Use the `tekton.dev/v1beta1` API version instead.
 //  (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
 
+* In {pipelines-title} 1.8, support for the `PipelineResource` custom resource (CR) has been removed. The `PipelineResource` CR was a Tech Preview feature and part of the `tekton.dev/v1alpha1` API.
+
 [id="known-issues-1-8_{context}"]
 == Known issues
 
-* There is currently a known issue: The `s2i-nodejs` pipeline cannot use the `nodejs:14-ubi8-minimal` image stream to perform source-to-image (S2I) builds. Doing so produces an `error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127` message.
+* There is currently a known issue: The `s2i-nodejs` pipeline cannot use the `nodejs:14-ubi8-minimal` image stream to perform source-to-image (S2I) builds. Using that image stream produces an `error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127` message.
```

Comment:
Is it necessary to say `There is a known issue:`? This sentence is a bullet point in the `Known issues` section after all.

Reply:
The [SSG release notes template](https://redhat-documentation.github.io/supplementary-style-guide/#release-notes) includes that wording. I think it is redundant in situations like this, where the removal notice is so close to the `== Known issues` heading. I think it makes more sense to keep that wording in the RHEL or OpenShift release notes where the removal statement might be far from the heading.

Reply:
Updated. Thanks.